### DISCUSSION TITLE
The Autistic Boy Who Became King

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#465] [Show and tell]
[The Autistic Boy Who Became King](https://youtu.be/FPKTfHB6vA8?si=T9rjFPFnLfL_NwOc) With the help of ToonSquid, I made my first real storytime animation. The rigging features may be basic right now, but I managed to create the illusion of Inverse Kinematics by animating the hands first, then drawing the arms. The more I do this kind of animation, the more workarounds I find, heck, I managed to find a way to have both pupils move with one control just recently.

**Comment by kevincreative** [comment]
Great story and well told and animated Dillon!
